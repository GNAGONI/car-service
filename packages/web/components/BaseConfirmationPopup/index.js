import BaseConfirmationPopup from './BaseConfirmationPopup';

export default BaseConfirmationPopup;
