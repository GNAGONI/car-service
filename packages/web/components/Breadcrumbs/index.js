import React from 'react';
import PropTypes from 'prop-types';
import { withRouter } from 'next/router';

import { capitalize } from 'lib/formatters';
import BreadcrumbItem from './BreadcrumbItem';

const Breadcrumbs = ({ breadcrumbs, router: { asPath } }) => {
  // getting array of all parent paths
  const routesItems = asPath.split('/').slice(1);
  // breadcrumb item with link to root should be always shown
  let breadcrumbsItems = [{ link: '/', name: 'Home', active: routesItems.length === 0 }];

  // if breadcrumbs not explicitly set, we parsing URL and making them out of it
  if (breadcrumbs.length === 0) {
    routesItems.forEach((value, index) => {
      const isLast = index + 1 === routesItems.length;
      const link = `/${routesItems.slice(0, index + 1).join('/')}`; // making link by concatenating previous routes items with current one
      const name = capitalize(value.split(/[-]/).join(' ')); // formatting kebab-case to Capitalized Case
      const active = isLast; // if current breadcrumb item is the last one, then it should be active
      breadcrumbsItems.push({ link, name, active });
    });
  } else {
    breadcrumbsItems = breadcrumbsItems.concat(breadcrumbs);
  }

  return (
    <nav className="nav-breadcrumb" aria-label="breadcrumbs">
      <ol className="breadcrumb">{breadcrumbsItems.map(BreadcrumbItem)}</ol>
    </nav>
  );
};

Breadcrumbs.propTypes = {
  breadcrumbs: PropTypes.arrayOf(
    PropTypes.shape({
      name: PropTypes.string,
      link: PropTypes.string,
      active: PropTypes.bool,
    }),
  ),
  router: PropTypes.shape({
    asPath: PropTypes.string,
  }),
};

Breadcrumbs.defaultProps = {
  router: {
    asPath: '',
  },
  breadcrumbs: [],
};

export default withRouter(Breadcrumbs);
