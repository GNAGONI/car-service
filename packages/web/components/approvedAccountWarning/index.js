import React, { useState } from 'react';
import cn from 'classnames';
import { CREATE_ACCOUNT, SIGN_IN } from 'constants/messages';
import CreateAccountPopup from 'components/approvedAccountWarning/createAccountPopup';
import SignInPopup from 'components/approvedAccountWarning//signInPopup';

const signinContent = `Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim.`;

const createContent = `Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim.`;

const ApprovedAccountWarning = () => {
  const [currentTab, setCurrentTab] = useState(SIGN_IN);

  return (
    <div className="popup popup-md popup-signin bg-light text-center">
      <header className="section-head bordered">
        <h2>
          You need to hold an <strong>approved account</strong> to be able to bid on this car
        </h2>
      </header>
      <div className="popup-content">
        <ul className="tabset">
          <li className={cn({ active: currentTab === SIGN_IN })}>
            <a onClick={() => setCurrentTab(SIGN_IN)} role="button" tabIndex="-1" onKeyUp={() => {}}>
              {SIGN_IN}
            </a>
          </li>
          <li className={cn({ active: currentTab === CREATE_ACCOUNT })}>
            <a onClick={() => setCurrentTab(CREATE_ACCOUNT)} role="button" tabIndex="-1" onKeyUp={() => {}}>
              {CREATE_ACCOUNT}
            </a>
          </li>
        </ul>
        <div className="tab-content">
          {currentTab === SIGN_IN ? (
            <SignInPopup textContent={signinContent} />
          ) : (
            <CreateAccountPopup textContent={createContent} />
          )}
        </div>
      </div>
    </div>
  );
};

export default ApprovedAccountWarning;
