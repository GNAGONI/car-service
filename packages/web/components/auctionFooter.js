import React, { PureComponent } from 'react';
import { Row, Col } from 'react-bootstrap';
import PropTypes from 'prop-types';
import CommonCollapse from 'components/carAuctionResults/commonCollapse';

export default class AuctionFooter extends PureComponent {
  static propTypes = {
    bottomBlockTitle: PropTypes.string,
    bottomBlockDescriptionLeft: PropTypes.string,
    bottomBlockDescriptionRight: PropTypes.string,
    bottomReadMoreBlockDescriptionLeft: PropTypes.string,
    bottomReadMoreBlockDescriptionRight: PropTypes.string,
  };

  static defaultProps = {
    bottomBlockTitle: '',
    bottomBlockDescriptionLeft: '',
    bottomBlockDescriptionRight: '',
    bottomReadMoreBlockDescriptionLeft: '',
    bottomReadMoreBlockDescriptionRight: '',
  };

  renderAlwaysShowElements = () => {
    const { bottomBlockTitle, bottomBlockDescriptionLeft, bottomBlockDescriptionRight } = this.props;

    return (
      <>
        {bottomBlockTitle && (
          <header className="section-head">{<h1 dangerouslySetInnerHTML={{ __html: bottomBlockTitle }} />}</header>
        )}
        <Row className="mb-3">
          <Col md={6} dangerouslySetInnerHTML={{ __html: bottomBlockDescriptionLeft }} />
          <Col md={6} dangerouslySetInnerHTML={{ __html: bottomBlockDescriptionRight }} />
        </Row>
      </>
    );
  };

  render() {
    const { bottomReadMoreBlockDescriptionLeft, bottomReadMoreBlockDescriptionRight } = this.props;

    return (
      <Row className="mb-3 mb-md-5 mt-3 mt-md-5">
        <Col>
          <CommonCollapse
            renderAlwaysShowElements={this.renderAlwaysShowElements}
            isShowReadMore={bottomReadMoreBlockDescriptionLeft || bottomReadMoreBlockDescriptionRight}
          >
            <Row>
              <Col md={6} dangerouslySetInnerHTML={{ __html: bottomReadMoreBlockDescriptionLeft }} />
              <Col md={6} dangerouslySetInnerHTML={{ __html: bottomReadMoreBlockDescriptionRight }} />
            </Row>
          </CommonCollapse>
        </Col>
      </Row>
    );
  }
}
