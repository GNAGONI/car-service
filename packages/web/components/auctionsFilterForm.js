import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import CommonSelector from './commonSelector/commonSelector';
import { THEMES } from './commonSelector/constans';

export default class AuctionsFilterForm extends PureComponent {
  state = {
    search: '',
    sortBy: null,
  };

  static propTypes = {
    sortByOptions: PropTypes.arrayOf(
      PropTypes.shape({
        label: PropTypes.string,
        value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
      }),
    ),
    onSubmit: PropTypes.func,
  };

  static defaultProps = {
    onSubmit: () => {},
    sortByOptions: [],
  };

  onSearch = e => {
    this.setState({ search: e.target.value });
  };

  onSelect = data => {
    this.setState({ sortBy: data });
  };

  onSubmit = e => {
    e.preventDefault();
    const { sortBy, search } = this.state;

    this.props.onSubmit({ sortBy: sortBy && sortBy.value, search });
  };

  render() {
    const { sortByOptions } = this.props;

    const { search, sortBy } = this.state;

    return (
      <form action="#" className="filter-form" onSubmit={this.onSubmit}>
        <div className="input-row">
          <div className="form-group">
            <strong className="title">Filter Results</strong>
            <div className="form-row">
              <span className="label-wrap">
                <label htmlFor="search">
                  <i className="icon-search" />
                </label>
              </span>
              <div className="input-holder">
                <div className="input-wrap">
                  <input
                    type="text"
                    id="search"
                    className="form-control"
                    placeholder="Search by Keywords"
                    onChange={this.onSearch}
                    value={search}
                  />
                </div>
                <div className="btn-holder">
                  <button type="button" className="btn btn-dark">
                    Search
                  </button>
                </div>
              </div>
            </div>
          </div>
          <div className="form-group">
            <strong className="title">Sort by:</strong>
            <div className="form-row">
              <span className="label-wrap">
                <label htmlFor="sort">
                  <i className="icon-filter" />
                </label>
              </span>
              <div className="input-holder">
                <div className="input-wrap">
                  <CommonSelector
                    options={sortByOptions}
                    placeholder="Filter by..."
                    value={sortBy}
                    onSelect={this.onSelect}
                    theme={THEMES.partnerPortal}
                  />
                </div>
                <div className="btn-holder">
                  <button type="button" className="btn btn-dark btn-apply">
                    Apply
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </form>
    );
  }
}
