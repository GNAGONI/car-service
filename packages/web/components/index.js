export { default as FormErrorField } from './FormErrorField';
