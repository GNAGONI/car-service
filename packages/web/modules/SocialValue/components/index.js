export { default as SocialValueHeader } from './SocialValueHeader';
export { default as SocialValuePartnersImpact } from './SocialValuePartnersImpact';
export { default as SocialValueArticle } from './SocialValueArticle';
