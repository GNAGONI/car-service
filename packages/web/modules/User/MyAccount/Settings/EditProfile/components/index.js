export { default as EditProfilePopup } from './EditProfilePopup';
