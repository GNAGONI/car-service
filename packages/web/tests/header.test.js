import React from 'react';
import renderer from 'react-test-renderer';
import { DefaultHeader } from '../components/Header';

it('renders correctly', () => {
  const tree = renderer.create(<DefaultHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
