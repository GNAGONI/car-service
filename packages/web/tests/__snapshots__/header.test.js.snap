// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  id="header"
>
  <div
    className="container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col header-wrap"
      >
        <div
          className="logo"
        >
          <a
            href="/car-auctions"
          >
            <img
              alt="Car.serviceÂ®"
              height="44"
              src=""
              width="216"
            />
          </a>
        </div>
        <nav
          className="navbar navbar-expand-xxl navbar-dark"
          id="nav"
        >
          <button
            aria-controls="navbarSupportedContent"
            aria-expanded="false"
            aria-label="Toggle navigation"
            className="navbar-toggler collapsed"
            data-target="#navbarSupportedContent"
            data-toggle="collapse"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </button>
          <div
            className="collapse navbar-collapse"
            id="navbarSupportedContent"
          >
            <div
              className="navbar-wrap"
            >
              <ul
                className="navbar-nav mr-auto"
              >
                <li
                  className="nav-item"
                >
                  <a
                    className="nav-link"
                    href="/car-auctions"
                  >
                    <span
                      className="icon-wrap"
                    >
                      <img
                        alt="Car auctions"
                        height="28"
                        src=""
                        width="27"
                      />
                    </span>
                    Car auctions
                  </a>
                  <ul>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                  </ul>
                </li>
                <li
                  className="nav-item"
                >
                  <a
                    className="nav-link"
                    href="#"
                  >
                    <span
                      className="icon-wrap"
                    >
                      <img
                        alt="image description"
                        height="22"
                        src=""
                        width="27"
                      />
                    </span>
                    Scrap my car
                  </a>
                  <ul>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                  </ul>
                </li>
                <li
                  className="nav-item"
                >
                  <a
                    className="nav-link"
                    href="#"
                  >
                    <span
                      className="icon-wrap"
                    >
                      <img
                        alt="Private number plates"
                        height="24"
                        src=""
                        width="26"
                      />
                    </span>
                    Private number plates
                  </a>
                  <ul>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                  </ul>
                </li>
                <li
                  className="nav-item"
                >
                  <a
                    className="nav-link"
                    href="#"
                  >
                    <span
                      className="icon-wrap"
                    >
                      <img
                        alt="Car news & reviews"
                        height="18"
                        src=""
                        width="27"
                      />
                    </span>
                    Car news & reviews
                  </a>
                  <ul>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                  </ul>
                </li>
                <li
                  className="nav-item"
                >
                  <a
                    className="nav-link"
                    href="#"
                  >
                    <span
                      className="icon-wrap"
                    >
                      <img
                        alt="Finance, insurance & more"
                        height="24"
                        src=""
                        width="20"
                      />
                    </span>
                    Finance, insurance & more
                  </a>
                  <ul>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                  </ul>
                </li>
                <li
                  className="nav-item"
                >
                  <a
                    className="nav-link"
                    href="#"
                  >
                    <span
                      className="icon-wrap"
                    >
                      <img
                        alt="Social value"
                        height="21"
                        src=""
                        width="20"
                      />
                    </span>
                    Social value
                  </a>
                  <ul>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Link to page here
                      </a>
                    </li>
                  </ul>
                </li>
              </ul>
            </div>
          </div>
        </nav>
        <ul
          className="option-links"
        >
          <li
            className="dropdown"
          >
            <a
              aria-expanded="false"
              aria-haspopup="true"
              className="dropdown-toggle"
              data-toggle="dropdown"
              href="#"
              id="dropdownMessage"
              role="button"
            >
              <span
                className="img-wrap"
              >
                <img
                  alt="image description"
                  height="24"
                  src=""
                  width="26"
                />
              </span>
              <span
                className="counter"
              >
                3
              </span>
            </a>
            <div
              aria-labelledby="dropdownMessage"
              className="dropdown-menu message-drop"
            >
              <div
                className="jcf-scrollable"
              >
                <ul
                  className="drop-list"
                >
                  <li
                    className="status-green"
                  >
                    <a
                      href="#"
                    >
                      <span
                        className="title"
                      >
                        Message Subject here
                      </span>
                      <span
                        className="message-text"
                      >
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor..
                         
                      </span>
                      <time
                        className="message-time"
                        dateTime="2011-01-12"
                      >
                        18th September @ 14:29
                      </time>
                    </a>
                  </li>
                  <li
                    className="status-green"
                  >
                    <a
                      href="#"
                    >
                      <span
                        className="title"
                      >
                        A larger message Subject here and here over two lines
                      </span>
                      <span
                        className="message-text"
                      >
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor..
                         
                      </span>
                      <time
                        className="message-time"
                        dateTime="2011-01-12"
                      >
                        18th September @ 14:29
                      </time>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <span
                        className="title"
                      >
                        A larger message Subject here and here over two lines
                      </span>
                      <span
                        className="message-text"
                      >
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor..
                         
                      </span>
                      <time
                        className="message-time"
                        dateTime="2011-01-12"
                      >
                        18th September @ 14:29
                      </time>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <span
                        className="title"
                      >
                        Message Subject here
                      </span>
                      <span
                        className="message-text"
                      >
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor..
                         
                      </span>
                      <time
                        className="message-time"
                        dateTime="2011-01-12"
                      >
                        18th September @ 14:29
                      </time>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <span
                        className="title"
                      >
                        Message Subject here
                      </span>
                      <span
                        className="message-text"
                      >
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor..
                         
                      </span>
                      <time
                        className="message-time"
                        dateTime="2011-01-12"
                      >
                        18th September @ 14:29
                      </time>
                    </a>
                  </li>
                </ul>
              </div>
              <div
                className="btn-holder"
              >
                <a
                  className="btn-drop"
                  href="#"
                >
                  View all messages
                </a>
              </div>
            </div>
          </li>
          <li
            className="dropdown"
          >
            <a
              aria-expanded="false"
              aria-haspopup="true"
              className="dropdown-toggle"
              data-toggle="dropdown"
              href="#"
              id="dropdownUser"
              role="button"
            >
              <span
                className="img-wrap"
              >
                <img
                  alt="image description"
                  height="27"
                  src=""
                  width="27"
                />
              </span>
              <span
                className="counter"
              >
                4
              </span>
            </a>
            <div
              aria-labelledby="dropdownUser"
              className="dropdown-menu user-drop"
            >
              <div
                className="jcf-scrollable"
              >
                <ul
                  className="drop-list"
                >
                  <li
                    className="status-green"
                  >
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="20"
                          src=""
                          width="27"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          You have a car 
                          <strong>
                            awaiting collection
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          10 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                  <li
                    className="status-green"
                  >
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="26"
                          src=""
                          width="21"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          You were the highest bidder on 
                          <strong>
                            Vauxhall Corsa 1.6 GTI
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          10 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                  <li
                    className="status-red"
                  >
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="25"
                          src=""
                          width="28"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          You have been outbid on the car 
                          <strong>
                            Citreon Saxo 1.4 RS
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          29 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                  <li
                    className="status-red"
                  >
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="21"
                          src=""
                          width="23"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Please pay for the vehicle, 
                          <strong>
                            Vauxhall Corsa 1.6 RS
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          40 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="25"
                          src=""
                          width="21"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          You have a scheduled vehicle for collection, in 
                          <strong>
                            2 hours
                          </strong>
                           at
                           
                          <strong>
                            PR1 1YH
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          40 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="20"
                          src=""
                          width="27"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          You have a car 
                          <strong>
                            awaiting collection
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          52 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="20"
                          src=""
                          width="27"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          You have a car 
                          <strong>
                            awaiting collection
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          52 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="20"
                          src=""
                          width="27"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          You have a car 
                          <strong>
                            awaiting collection
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          52 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="20"
                          src=""
                          width="27"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          You have a car 
                          <strong>
                            awaiting collection
                          </strong>
                        </p>
                        <span
                          className="text-small"
                        >
                          52 mins ago
                        </span>
                      </div>
                    </a>
                  </li>
                </ul>
              </div>
            </div>
          </li>
          <li
            className="dropdown"
          >
            <a
              aria-expanded="false"
              aria-haspopup="true"
              className="dropdown-toggle"
              data-toggle="dropdown"
              href="#"
              id="dropdownSetting"
              role="button"
            >
              <span
                className="img-wrap"
              >
                <img
                  alt="image description"
                  height="23"
                  src=""
                  width="23"
                />
              </span>
            </a>
            <div
              aria-labelledby="dropdownSetting"
              className="dropdown-menu setting-drop"
            >
              <div
                className="jcf-scrollable"
              >
                <ul
                  className="drop-list"
                >
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="15"
                          src=""
                          width="14"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Edit profile
                        </p>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="22"
                          src=""
                          width="20"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Change password
                        </p>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="22"
                          src=""
                          width="28"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Edit areas & car makes
                        </p>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="16"
                          src=""
                          width="19"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Notifications
                        </p>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="19"
                          src=""
                          width="32"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Edit Vehicle preferences
                        </p>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="17"
                          src=""
                          width="22"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Subscriptions
                        </p>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="16"
                          src=""
                          width="19"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Notifications
                        </p>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="19"
                          src=""
                          width="32"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Edit Vehicle preferences
                        </p>
                      </div>
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      <div
                        className="icon-wrap"
                      >
                        <img
                          alt="image desctiption"
                          height="17"
                          src=""
                          width="22"
                        />
                      </div>
                      <div
                        className="text-holder"
                      >
                        <p>
                          Subscriptions
                        </p>
                      </div>
                    </a>
                  </li>
                </ul>
              </div>
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
</header>
`;
